src/context/cdlist.h

```diff
@@ -163,7 +168,8 @@ class CDList : public ContextObj {
         throw std::bad_alloc();
       }
       std::memcpy(newList, d_list, sizeof(T) * d_sizeAlloc);
```

Comment:
Are we checking somewhere that `T` is trivially copyable?

Reply:
No, there is actually a long-standing PR #2149 that switches to an `std::vector` to avoid those issues. I should take another look...